cross-images/libpq.sh

```diff
@@ -0,0 +1,47 @@
+set -ex
+
+main() {
+    local version=9.6.9
+    local triple=$1
+
+    local dependencies=(
+        ca-certificates
+        curl
+        m4
+        make
+        perl
+    )
+
+    # NOTE cross toolchain must be already installed
+    apt-get update
+    local purge_list=()
+    for dep in ${dependencies[@]}; do
+        if ! dpkg -L $dep; then
+            apt-get install --no-install-recommends -y $dep
+            purge_list+=( $dep )
+        fi
+    done
+
+    td=$(mktemp -d)
+
+    pushd $td
+    curl -kfLO https://ftp.postgresql.org/pub/source/v$version/postgresql-$version.tar.gz
```

Comment:
Adding some comment about what all this is about would be awesome.

Reply:
I have added comments. Also, it will need refactoring base on how this goes https://github.com/quickwit-inc/quickwit/issues/715